manifests/v/VoyagerX/Vrew/0.8.17/VoyagerX.Vrew.installer.yaml

```
@@ -0,0 +1,20 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: VoyagerX.Vrew
+PackageVersion: 0.8.17
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
```

```suggestion
  InstallerType: nullsoft
```
exe's should always contain some kind of InstallerSwitches, **Universal Silent Switch Finder** detects this is as Nullsoft which from my testing is true, now it won't open after installation.